crypto/encoding/amino/amino.go

```diff
@@ -60,11 +60,20 @@ func RegisterAmino(cdc *amino.Codec) {
 		secp256k1.PrivKeyAminoName, nil)
 }
 
+// RegisterKeyType registers an external key type to allow decoding it from bytes
+func RegisterKeyType(o interface{}, name string) {
```

Comment:
Would be great to have a test case for it.

Reply:
Will do